### PR TITLE
resolving 5 Jira DOC tasks.

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/about_supported_platforms.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ These are the minimum platform specifications for running the 128T Networking Pl
 
 ### Router
 
-* 4C Intel x86 processor, single socket
+* Intel x86 processor, single socket
```
Comment (MichaelBaj) [inline docs/about_supported_platforms.md:130]: Why did you remove the 4 core requirement?

File: docs/about_supported_platforms.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ Multi-socket platforms are not compatible with the 128T software when run as a r
 
 ### Conductor
 
-* 4C Intel processor
+* Intel x86 processor
```
Comment (MichaelBaj) [inline docs/about_supported_platforms.md:145]: Why did you remove the 4 core requirement?

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/about_supported_platforms.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/about_supported_platforms.md:130]: In DOCS-236, Dave requested it be changed to "Intel x86 processor". I have never see 4 Core referred to as 4C, and apparently neither had he. I will change it to 4 Core.

File: docs/about_supported_platforms.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/about_supported_platforms.md:145]: Same as above.

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**